### DISCUSSION TITLE
Setting up a client on a router with Padavan firmware

### DISCUSSION THREAD

**Discussion by Frostoman** [pivpn/pivpn#1717] [Support]
Hello. Please help me set up the client on the router. The server is working, and there is a connection through the OpenVPN application on my phone and PC. But I couldn't set up the client on the router. I can't import the configuration file into the router, so I have to do everything manually. Maybe I did something wrong in the settings...

It connects to paid VPN servers, but not to mine :(

The certificates and keys were cut off for security reasons.

![01](https://user-images.githubusercontent.com/32976590/232894161-9a4f9700-ad41-465b-8ce0-7e6884f17f80.png)
![02](https://user-images.githubusercontent.com/32976590/232894177-7ea59282-51b3-4ae6-81f6-5eb934156627.png)

**Comment by orazioedoardo** [comment]
Disable data compression. Check the log on the Raspberry Pi at `/var/log/openvpn.log`

**Reply by Frostoman** [reply]
Thank for advice. I will try it today
